src/main/java/com/github/manolo8/darkbot/utils/LogUtils.java

```diff
@@ -111,4 +133,62 @@ public void println(String string) {
             super.println(string);
         }
     }
+
+    public static class MultiOutputStream extends OutputStream {
+        private final OutputStream[] outputStreams;
+
+        public MultiOutputStream(OutputStream... outputStreams) {
+            this.outputStreams = outputStreams;
+        }
+
+        @Override
+        public void write(int b) throws IOException {
+            try {
+                for (OutputStream out : outputStreams)
+                    out.write(b);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
```

Comment:
these try-catch + for need to be reversed to for + try

Otherwise one output stream failing means the other ones never get the data